HomeWork/src/main/java/ionio/Main.java

```diff
@@ -0,0 +1,28 @@
+package ionio;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.Properties;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+
+FileLoggerConfiguration[] loggerCfg = {
```

Comment:
How did you test it? This doesn't look like a working code, it is just an array.